Makefile.am

```diff
@@ -403,14 +320,16 @@ libpriv_db_assets_la_SOURCES = \
 			src/db/asset_general.cc \
 			src/db/asset_general.h
 
-libpriv_db_assets_la_LIBADD = ${LIBCZMQ_LIBS}
+libpriv_utils_la_LDFLAGS = ${CXXTOOLS_LIBS} -ltntdb
```

Comment:
So here - add `${LIBCZMQ_LIBS}`

Note to replace `-ltntdb` by `${TNTDB_LIBS}` when respective PR is merged... oh it will pop up among merge conflicts there anyway...